### ISSUE TITLE
WPF: Databinding & multiple views out of sync

### ISSUE THREAD

**Issue by ChrisonSimtian** [helix-toolkit/helix-toolkit#1402]
Hey,

I have one master control containing a helix toolkit (WPF) control that feeds in with the same ViewModel from multiple views.
Lately I've experienced some weird side effects of this construct, where this control seems to be out of synch with the other views, even though the underlying viewmodel is exactly the same.
Is there anything within the helix toolkit control, that delays databinding?
For me the effect is mostly that the currently active page doesnt show my 3D model. The other 2 views show it though and as soon as I navigate back the first page shows it as well but in a different orientation.

**Comment by chrkon** [comment]
_Just as additional Info:_
I am not sure, but I think I have seen this behaviour on a ".net core WPF with SharpDX" project, too. At the end I thought, that I had made something wrong, so I have not searched the bug in the Helix Toolkit components.
We have up to 4 ViewPort controls which are based on the same Model.

**Comment by eriove** [comment]
We have four view ports as well but haven't seen this so far. We are multi targeting both .NET Core and .NET Framework with SharpDX.

**Comment by chrkon** [comment]
Hi @ChrisonSimtian, 
it is not easy for me to understand the architecture you use. Could you describe it more detailed or is it possible to get some code to reproduce this behaviour?

You said you have one master control View (Helix Toolkit WPF) which is out of sync with the ViewModel data. You mentioned that two other views show the data correctly. Are these views also Helix Toolkit Viewports?

And in the last sentence you said if you navigate back to the first page, the _(i assume)_ master control shows the ViewModel data in a wrong orientation.

Does this mean, the master control view is not visible until you switch back to the first page?

Have you tried to update the visuals on the activation event of the page. I am not sure whether the Helix Viewport is updated while it is not visible.

**Comment by holance** [comment]
@ChrisonSimtian Make sure you are not sharing the same camera or camera parameters with all of your viewports.

**Comment by ChrisonSimtian** [comment]
> @ChrisonSimtian Make sure you are not sharing the same camera or camera parameters with all of your viewports.

Hey,
I thought your advice fixed the issue but today when I implemented a prototype (no shared objects with any other of the helix views we have) I got the same scenario again:

1. I am on View A which holds a HelixToolkit:ViewPort
2. I load my mesh into it
3. The view doesnt show anything
4. I go to a different view (View B), which generates another mesh (based on the same cloudpoint, so it shows the same information)
5. View B shows the model
6. I switch back to View A
7. Suddenly View A shows the model
